### DISCUSSION TITLE
Publish encrypted notes

### DISCUSSION THREAD

**Discussion by nicolasduteil** [standardnotes/forum#2502] [Suggestions]
Hi.
Would it be possible to have a way to publish a note using which would require a password ?. Using JS encryption like what http://www.vincentcheung.ca/jsencryption/ does (not saying this is the best way to do it ;)). This would be useful when you want to restrict access the to published content.

**Comment by atmoio** [comment]
Hmm..I suppose it could be possible. Probably not something I'll work on soon, but as more people start using Listed. But I like the idea. I've made a note to check it out to see what it involves.

**Comment by JeremyMusick** [comment]
NotesNook has this, I'd love to see this feature on Standard Notes.

**Comment by xthrk** [comment]
I would love this too. Like today's Listed Action "Publish to Private Link" but with a password that you can choose yourself. Great would also be additional options like a self-destructing link after x days or y calls. So that the link is deleted after, for example, 5 days. Or that the link can be called only 1 time.

**Comment by myreli** [comment]
While sharing encrypted notes directly from the app isn't currently possible, I will register the alternatives here for those wondering: 

- [Listed](https://listed.to): publish notes directly from SN **unencrypted** 
- [FileSend](https://filesend.standardnotes.com): share files manually **encrypted**

I believe the suggestion now is to bridge FileSend from StandardNotes (https://github.com/standardnotes/forum/discussions/2623). 🤔
